src/utils/object.js

```diff
@@ -205,6 +207,8 @@ export function nonEmptyValueKeys(obj) {
     } else if (!!val || val === false || val === 0) {
       return key;
     }
+
+    return null;
```

Comment:
again as opposed to an implicit `return undefined`?

Are you going to submit a PR to dashboard?

Reply:
Commit dropped :)